oauth-common/src/main/java/io/strimzi/kafka/oauth/metrics/JmxMetrics.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2017-2021, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.oauth.metrics;
+
+import io.strimzi.kafka.oauth.common.HttpException;
+
+import javax.net.ssl.SSLException;
+import java.io.IOException;
+import java.net.ConnectException;
+import java.net.URI;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class JmxMetrics {
```

Comment:
1. Javadoc
2. What's JMX about this class? It seems pretty independent of JMX to me?

Reply:
True, after refactoring all the JMX parts were removed, but the class name remained.

Reply:
Added javadoc, renamed class to MetricsUtil.